samcli/hook_packages/terraform/hooks/prepare/makefile_generator.py

```diff
@@ -88,6 +89,12 @@ def generate_makefile(
     )
     shutil.copy(copy_terraform_built_artifacts_script_path, output_directory_path)
 
+    # copy zip.py script into output directory
+    ZIP_UTILS_MODULE_script_path = os.path.join(
+        Path(os.path.dirname(__file__)).parent.parent.parent.parent, "local", "lambdafn", ZIP_UTILS_MODULE
```

Comment:
the `.parent.parent.parent.parent` seems a bit weird. Is there a better way of doing this?

Reply:
I defined the samcli root path in a separate variable to make it more readable